app/src/main/res/layout/fragment_private_homescreen.xml

```diff
@@ -54,7 +54,7 @@
         android:id="@+id/pm_home_fake_input"
         style="@style/FakeInput"
         layout="@layout/fragment_homescreen_item_fake_input"
-        android:layout_width="match_parent"
+        android:layout_width="306dp"
```

Comment:
Is this change per spec? If so, should we align this with normal browsing too? (Normal browsing is match_parent)